packages/compass-crud/src/stores/crud-store.spec.ts

```diff
@@ -190,6 +196,7 @@ describe('store', function () {
           logger: createNoopLoggerAndTelemetry(),
           favoriteQueryStorageAccess: compassFavoriteQueryStorageAccess,
           recentQueryStorageAccess: compassRecentQueryStorageAccess,
+          fieldStoreService: mockFieldStoreService,
```

Comment:
If we're not testing it here, maybe we can provide some fallbacks for test env so that we don't need to override it (no strong preference for this though)

Reply:
We are calling the plugin activate directly here so the locator does not come into play for this part which is why I think we would still need to pass the mocked service.

Reply:
Ah, didn't notice that, then it's kinda unavoidable, yeah